### ISSUE TITLE
Feature Request: Hide seen toots

### ISSUE THREAD

**Issue by pabloscloud** [pkreissel/foryoufeed#5]
I want an option to hide seen posts.

If this is hard to implement an option to hide toots that I've interacted with would be cool as an alternative.

**Comment by pkreissel** [comment]
Yes I was thinking about that too. One easy to implement heuristic would be to remember the last time you visited the feed and then show stuff after that date only. Would also shorten loading time. Another option would be to increase the penalty for older posts if the feed notices that you visit it more frequently.

**Comment by pabloscloud** [comment]
Sounds good to me. Either way

**Comment by pkreissel** [comment]
Implemented with [this](https://github.com/pkreissel/fedialgo/commit/1c29492a272f70e25fb2dd0d1e182edf5661f2d0). Tell me if it works sufficiently well.

**Comment by pabloscloud** [comment]
You're fast! Still trying it out!
